source/system-administrators/activities/reindexing-content.rst

```diff
@@ -25,6 +25,57 @@ Clear index through API
 
 To delete any existing content in the index through API, send the following CURL command:
 
+.. WARNING::
```

Comment:
Shouldn't we add a hint and a cross-ref to the production process here so people know there is another way to do this for live envs?